src/base/numbers.c

```diff
@@ -74,3 +74,42 @@ int _getHexString(char *buffer, unsigned long hexnumber, bool use_capital) {
     }
     return size;
 }
+
+// Unsigned
+int itoa(char *buffer, unsigned long number, int base, bool use_capital) {
+	if (base < 1 || base > 36) return 0;
+	char *pointer, *pointerbase;
+	int mod;
+	int size = 0;
+	char letter_base = 'a';
+	if (use_capital) {
+		letter_base = 'A';
+	}
+	pointer = buffer;
+	pointerbase = buffer;
```

Comment:
I would also suggest an empty line after this one, to show separate between initializing variables and when the function begins to do real work.

Reply:
You are talking about a newline after pointerbase = buffer; correct?

Reply:
Yeah :)
Looks like you already added it, thanks.